commands/verify/verify.go

```diff
@@ -87,6 +88,87 @@ func verifySingleCase(expectedFile, actualFile, query string) error {
 	return nil
 }
 
+// concurrentErrors store errors that occurred when verifying cases in goroutines.
+type concurrentErrors struct {
+	errs  *multierror.Error
+	mutex sync.Mutex
+	count int
+}
+
+// verifyInfo contains necessary information about verification
+type verifyInfo struct {
+	retryCount int
+	interval   time.Duration
+	failFast   bool
+}
+
+func concurrentSafeErrAppend(concurrentError *concurrentErrors, err error) {
+	concurrentError.mutex.Lock()
+	concurrentError.errs = multierror.Append(concurrentError.errs, err)
+	concurrentError.mutex.Unlock()
+}
+
+func concurrentSafeCountLess(concurrentError *concurrentErrors) {
+	concurrentError.mutex.Lock()
+	concurrentError.count--
+	concurrentError.mutex.Unlock()
+}
+
+func check(stopChan chan struct{}, concurrentErrors *concurrentErrors) error {
+	for {
+		if concurrentErrors.count == 0 {
+			break
+		}
+		_, ok := <-stopChan
```

Comment:
It will be blocked here if all concurrent cases success. Maybe you can refer to [this](https://stackoverflow.com/questions/45500836/close-multiple-goroutine-if-an-error-occurs-in-one-in-go).

Reply:
Thanks! I will learn it !